packages/generate/src/syntax/funcops.ts

```diff
@@ -127,9 +127,15 @@ export function $resolveOverload(
   throw new Error(
     `No function overload found for ${
       funcName.includes("::")
-        ? `'e.${funcName.split("::")[1]}()'`
+        ? `'e.${funcName.split("::").join(".")}()'`
```

Comment:
Supports submodules including `ext::pgvector` in this case. It was printing `e.pgvector()` instead of `e.ext.pgvector.cosine_distance()`.

Note: we're not dropping `std` and `math` here any more 🤔

Reply:
Good catch. I guess the previous logic was to strip out the `std` or `default` as they get aliased into the top level and we didn't have nested modules, but even that's broken because of the `cal` and `math` modules (and any non default user modules) 🤦.

Reply:
Yeah, I couldn't quickly think of a maintainable way to update this to still remove `std` and `default` other than hardcoding a filter in here, but maybe that's fine?

Reply:
Yeah, I think it's fine to always show the full module + function name in error messages. You can probably also shadow the top level aliases of the stdlib functions anyway, and I think it's not worth trying to handle all that correctly just for an error message.

Reply:
Actually, I wonder if it's even worth trying to typescript-ify the function names at all 🤔 I think it would be maybe more clear if the error message was pointing to the actual EdgeQL function that was called rather than trying to construct what it probably looks like at the call site? In other words, just use `funcName` here. Thoughts?